### ISSUE TITLE
Only two letters can be typed in when searching for entry

### ISSUE THREAD

**Issue by tntturtl** [bitwarden/browser#673]
Hi everyone,

I installed Bitwarden extension for Edge browser. 
When I try to search for something in my vault I can only type first two letters and searching begins. Once searching is finished remaining letters of a word I typed in are displayed.

Any clues what could be wrong with the plugin or is it web browser?

**Comment by lhorace** [comment]
I disagree, I have come this issue with Chrome and Firefox. I've been reading your tickets, is your Target is MacOS?
